volatility3/framework/plugins/windows/vadinfo.py

```diff
@@ -196,11 +196,31 @@ def vad_dump(
 
         return file_handle
 
-    def _generator(self, procs):
+    def _generator(self, procs: List[interfaces.objects.ObjectInterface]) -> Generator[
+        Tuple[
+            int,
+            Tuple[
+                int,
+                str,
+                format_hints.Hex,
+                format_hints.Hex,
+                format_hints.Hex,
+                str,
+                str,
+                int,
+                int,
+                format_hints.Hex,
+                str,
+                str,
+            ],
+        ],
+        None,
+        None,
+    ]:
         kernel = self.context.modules[self.config["kernel"]]
         kernel_layer = self.context.layers[kernel.layer_name]
 
-        def passthrough(_: interfaces.objects.ObjectInterface) -> bool:
+        def passthrough(x: interfaces.objects.ObjectInterface) -> bool:
```

Comment:
Not sure why this needs naming, but ok?

Reply:
mypy is annoying otherwise :|

Reply:
Hehehe, yeah, you'll find full type checking does that (imagine trying it a few years ago when I started!)  5;P